media/js/src/assetDetail/AssetDetail.jsx

```diff
@@ -190,8 +190,12 @@ export default class AssetDetail extends React.Component {
         }
     }
 
-    onPlayerPlay() {
-        this.setState({playing: true});
+    onPlayToggle() {
+        let newState = true;
+        if (this.state.playing) {
+            newState = false;
+        }
+        this.setState({playing: newState});
```

Comment:
Maybe shorten this to:  
`this.setState({playing: this.state.playing});`

Reply:
Oh yeah - I assume you mean: `this.setState({playing: !this.state.playing});`

Good idea.

Reply:
Der, yep. :-)